src/com/itacademy/eshop/product/Product.java

```diff
@@ -35,6 +39,15 @@ public ArrayList<Review> getReviews() {
 
     public void addReview(Review review) {
```

Comment:
your method should do one thing and now you add a review and add some info. that info should be provided either via setter or in constructor